### ISSUE TITLE
Getting CSS values in JavaScript

### ISSUE THREAD

**Issue by Kid-42** [Kid-42/The-Kids-stuff#1]
I'm creating a simple puzzle game. (full credit goes to Kenny Yip whose code I'm trying to modify  https://www.youtube.com/watch?v=S6GNtMGNcUE ) I want to determine when all the pieces are in the right place.  It has the following in a .css file, where board is the target and pieces are where the images start out:
![image](https://github.com/user-attachments/assets/7032d11d-a56d-47fe-abb8-04ebc29b7ac9)

In my .js file I push images into these areas using
       document.getElementById("board").append(tile);
       document.getElementById("pieces").append(tile);
Where tile is an image.

Later in the .js file I swap the images in the file based upon where the image is dropped.  "this" gives me the images.  But I can't tell where in the board something is dropped.  I'd like to run though my board finding out what is in it.

Bottom line, how can I tell what's in my board??
